torch/_decomp/__init__.py

```diff
@@ -274,6 +274,8 @@ def core_aten_decompositions() -> Dict[OpOverload, Callable]:
             aten.mish_,
             aten.mse_loss,
             aten.mse_loss_backward,
+            aten.multi_margin_loss,
+            aten.multi_margin_loss_backward,
```

Comment:
not anymore

Reply:
This is required for this test to pass:

```
 pytest -rfxs test/test_decomp.py -k test_aten_core_operators -v
```

Reply:
@nkaretnikov rerun the test with EXPECTTEST_ACCEPT=1